components/StandardDialog.qml

```diff
@@ -152,6 +152,7 @@ Rectangle {
             MoneroComponents.StandardButton {
                 id: cancelButton
                 text: qsTr("Cancel") + translationManager.emptyString
+                visible: root.cancelVisible
```

Comment:
Doesn't it loop like this? `root.cancelVisible` is an alias to this line.

Reply:
Yep, it does. Fixed